neqo-qpack/src/stats.rs

```diff
@@ -0,0 +1,14 @@
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// Tracking of some useful statistics.
+
+#[derive(Default, Debug)]
+/// `QPack` statistics
+pub struct Stats {
+    pub dynamic_table_inserts: u64,
+    pub header_blocks_ref_dynamic_table: u64,
```

Comment:
Perhaps this would be clearer as `dynamic_table_references` with a comment explaining what the number represents.